NOTICE.txt

```diff
@@ -31,3 +31,29 @@ bring it to our attention. Post an issue or email us:
 The attached notices are provided for information only.
 
 ------------------------------------------------------------------------------
+Copyright (c) 2012 - 2017, Lars Bilke
+
+Boost Software License - Version 1.0 - August 17th, 2003
```

Comment:
Let's make sure this is OK to use.

Reply:
got confirmation from Angela (from legal) that Boost is fine

Reply:
We might want this to be in the cgmanifest.

Reply:
added cgmanifest